openff/toolkit/typing/engines/smirnoff/parameters.py

```diff
@@ -2813,10 +2814,47 @@ def to_dict(
 
     cutoff = ParameterAttribute(default=9.0 * unit.angstroms, unit=unit.angstrom)
     switch_width = ParameterAttribute(default=1.0 * unit.angstroms, unit=unit.angstrom)
-    method = ParameterAttribute(
-        default="cutoff", converter=_allow_only(["cutoff", "PME"])
+    periodic_method = ParameterAttribute(
+        default="cutoff", converter=_allow_only(["cutoff", "no-cutoff"])
```

Comment:
(not blocking) Darn... I was gonna request that we also continue supporting `"PME"` (just to store in-memory, even if it raises a NotImplementedError when folks try to use it), but without knowing whether it'll be called like "Ewald3D" or "PME" or "something-like-conducting-boundary", it would do more harm than good to shim it in. So I now realize that, when we start doing LJPME studies we'll need to start with a SMIRNOFF EP, then TK and IC releases, and then the studies can begin.

Reply:
Thanks for the leeway here - I'm not thrilled to reduce feature coverage (even though in this case it's unimplemented and not tested) but I think it's the right approach.

Hopefully LJPME is something we can get into the spec in short time, and hopefully it's only a small lift after 0008!